content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -51,6 +51,46 @@ The hint is then stored in the Topology Manager for use by the *Hint Providers*
 
 Support for the Topology Manager requires `TopologyManager` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) to be enabled. It is enabled by default starting with Kubernetes 1.18.
 
+### Topology Manager Scopes
+
+The Topology Manager can deal with the alignment of resources in a couple of distinct scopes:
+
+* `container` (default)
+* `pod`
+
+Either option can be selected at a time of the kubelet startup, with `--topology-manager-scope` flag.
+

```

Comment:
I would probably combine the Scope and Policy sections into 1 (i.e. `Topology Manager Scopes and Policies`). That way you can have a quick intro into what both of them are and how they interact before diving into the details of them.

Reply:
Thanks, I will prepare such an update today.

Reply:
I think after this is addressed then this is good to go(?)

Reply:
I hope so. I won't change any content here, only rearrange stuff.

Reply:
@irvifa @klueska 
I grouped scope and policies in the common section (## Topology Manager Scopes and Policies) and added a short description for it.